### PR TITLE
more prominent gitter channel plug

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by mcovarr** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ A [Workflow Management System](https://en.wikipedia.org/wiki/Workflow_management
 
 <!---toc start-->
 
-* [Mailing List](#mailing-list)
+* [How To Reach us](#how-to-reach-us)
```
Comment (mcovarr) [inline README.md]: cap 'us'

Comment (cjllanwarne) [inline README.md]: cap UK?

**Comment by cjllanwarne** [comment]
:+1:

**Pull request merged**